### PR TITLE
Added client operation for get query with parameters

### PR DESCRIPTION
**Pull request by Anipik**
GetQueryWithParameters:- 
- the first parameter is always of the maxContentLength. The reason behind this that http2 was failing sometimes when the length of the content is around 8000 and http 1.1 was stable for that. i will create an issue and post the call stack there.
- The number of parameters can be between 0 and 100 .

### CODE REVIEW

File: src/System.Net.Http/tests/StressTests/HttpStress/Program.cs [hunk 1]
```diff
@@ -158,6 +159,19 @@ void ValidateContent(string expectedContent, string actualContent)
                 }
             }),
 
+            ("GET Parameters",
+            async ctx =>
+            {
+                Version httpVersion = ctx.GetRandomVersion(httpVersions);
+                (string queryString, string expectedString) variables = GetGetQueryParameters(contentSource, ctx);
```
Comment (eiriktsarpalis) [inline src/System.Net.Http/tests/StressTests/HttpStress/Program.cs]: nit: a bit redundant maybe? Can do `(string x, string y) = GetGetQueryParameters(...)`

File: src/System.Net.Http/tests/StressTests/HttpStress/Program.cs [hunk 2]
```diff
@@ -586,6 +613,24 @@ void Increment(ref long counter)
...
+    {
+        StringBuilder queryString = new StringBuilder();
+        queryString.Append($"?Var{0}={contentSource}");
+        string expectedString = contentSource;
+
+        int num = clientContext.GetRandomInt(100);
+
+        for (int i = 1; i < num; i++)
+        {
+            string vari = clientContext.GetRandomSubstring(contentSource);
```
Comment (eiriktsarpalis) [inline src/System.Net.Http/tests/StressTests/HttpStress/Program.cs]: My only concern here is that the query string can end up being _very_ large, bounded by `100 * maxContentLength`. Should maybe be less than `maxContentLength` or some other size parameter?

Comment (eiriktsarpalis) [reply src/System.Net.Http/tests/StressTests/HttpStress/Program.cs]: Maybe renaming `maxContentLength` to something nondescript like `maxSize`?

File: src/System.Net.Http/tests/StressTests/HttpStress/Program.cs [hunk 3]
```diff
@@ -436,6 +450,19 @@ void ValidateContent(string expectedContent, string actualContent)
                             }
                         }
                     });
+                    endpoints.MapGet("/variables", async context =>
+                    {
+                        string queryString = context.Request.QueryString.Value;
+                        var nameValueCollection = HttpUtility.ParseQueryString(queryString);
```
Comment (stephentoub) [inline src/System.Net.Http/tests/StressTests/HttpStress/Program.cs]: Nit: `var` => whatever type it is

File: src/System.Net.Http/tests/StressTests/HttpStress/Program.cs [hunk 4]
```diff
@@ -586,6 +613,24 @@ void Increment(ref long counter)
...
+        StringBuilder queryString = new StringBuilder();
+        queryString.Append($"?Var{0}={contentSource}");
+        string expectedString = contentSource;
+
+        int num = clientContext.GetRandomInt(100);
+
+        for (int i = 1; i < num; i++)
+        {
+            string vari = clientContext.GetRandomSubstring(contentSource);
+            expectedString += vari;
```
Comment (stephentoub) [inline src/System.Net.Http/tests/StressTests/HttpStress/Program.cs]: Even for a test app this is a lot of unnecessary string allocation.  A StringBuilder here would be good as well.

File: src/System.Net.Http/tests/StressTests/HttpStress/Program.cs [hunk 5]
```diff
@@ -44,6 +45,7 @@ public static void Main(string[] args)
         cmd.AddOption(new Option("-aspnetlog", "Enable ASP.NET warning and error logging.") { Argument = new Argument<bool>("enable", false) });
         cmd.AddOption(new Option("-listOps", "List available options.") { Argument = new Argument<bool>("enable", false) });
         cmd.AddOption(new Option("-seed", "Seed for generating pseudo-random parameters for a given -n argument.") { Argument = new Argument<int?>("seed", null)});
+        cmd.AddOption(new Option("-p", "Max number of query paramters for a request.") { Argument = new Argument<int>("queryParameters", 1) });
```
Comment (eiriktsarpalis) [inline src/System.Net.Http/tests/StressTests/HttpStress/Program.cs]: paramters -> parameters

**Pull request merged**